src/embind/emval.js

```diff
@@ -448,7 +453,9 @@ var LibraryEmVal = {
     params.push(functionBody);
     var invokerFunction = new_(Function, params).apply(null, args);
 #endif
-    return __emval_addMethodCaller(invokerFunction);
+    returnId = __emval_addMethodCaller(invokerFunction);
```

Comment:
missing var here?

Reply:
`var returnId` is already declared on line 393 above https://github.com/emscripten-core/emscripten/pull/15561/files#diff-d144fb80a2276e9f98227198d34846fde9d73a6a6963a5ad768c6c69f776e490R393-R394